test/e2e/components_upgrade_test.go

```diff
@@ -0,0 +1,71 @@
+// +build e2e
+
+package e2e
+
+import (
+	"path/filepath"
+
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/semver"
+	releasev1alpha1 "github.com/aws/eks-anywhere/release/api/v1alpha1"
+	"github.com/aws/eks-anywhere/test/framework"
+)
+
+const (
+	prodReleasesManifest = "https://anywhere-assets.eks.amazonaws.com/releases/eks-a/manifest.yaml"
+	latestReleasePath    = "bin/latest-release"
+	releaseBinaryName    = "eksctl-anywhere"
+)
+
+func getLatestMinorReleaseFromMain(test *framework.ClusterE2ETest) (binaryPath string) {
+	reader := cluster.NewManifestReader()
+	test.T.Logf("Reading prod release manifest %s", prodReleasesManifest)
+	releases, err := reader.GetReleases(prodReleasesManifest)
+	if err != nil {
+		test.T.Fatal(err)
+	}
+
+	var latestRelease *releasev1alpha1.EksARelease
+	for _, release := range releases.Spec.Releases {
+		if release.Version == releases.Spec.LatestVersion {
+			latestRelease = &release
+			break
+		}
+	}
+
+	if latestRelease == nil {
+		test.T.Fatalf("Releases manifest doesn't contain latest release %s", releases.Spec.LatestVersion)
+	}
+
+	latestReleaseBinaryFolder := filepath.Join(latestReleasePath, latestRelease.Version)
+	latestReleaseBinaryPath := filepath.Join(latestReleaseBinaryFolder, releaseBinaryName)
+
+	return latestReleaseBinaryPath
```

Comment:
what is this path referring to? Are you planning on adding some code to download the binaries to those paths?

Reply:
Yeah, I was planning to add that in a separate PR when we add the actual tests, but I think it'll be best to take this out for now until we solidify the framework for these e2e tests.